src/ray/protobuf/node_manager.proto

```diff
@@ -335,6 +335,14 @@ message NotifyGCSRestartRequest {}
 
 message NotifyGCSRestartReply {}
 
+message GetTaskResultRequest {
+  bytes task_id = 1;
+}
+
+message GetTaskResultReply {
+  optional RayException failure_cause = 1;
```

Comment:
Can we use `ErrorType` instead of `RayException` in CoreWorker and Raylet?

Seems `RayException` is meant for exceptions in the language frontend, so we should try to keep the backend agnostic to these if possible.